prime-router/metadata/fhir_mapping/hl7/resource/ServiceRequest.yml

```diff
@@ -50,11 +50,12 @@ identifier_2b:
   vars:
     valueIn: OBR.3.1
     systemCX: OBR.3.2
+    universalId: OBR.3.3
+    universalIdType: OBR.3.4
     valueInORC: ORC.3.1
   constants:
     system: "http://terminology.hl7.org/CodeSystem/v2-0203"
     code: "FILL"
-    display: "Filler Identifier"
```

Comment:
Why was the display removed?

Reply:
Same as [previous comment](https://github.com/CDCgov/prime-reportstream/pull/8017/files/6e7f4a87be69be0f0c71a2cb8d762220b04c1188#r1082812908).